src/main/java/nl/hsac/fitnesse/fixture/util/selenium/SeleniumHelper.java

```diff
@@ -600,6 +600,38 @@ public void dragAndDropToOffsetXY(WebElement element, Integer xOffset, Integer y
         getActions().dragAndDropBy(element, xOffset, yOffset).build().perform();
     }
 
+    /**
+     * Simulates a drag of element to destination offsets calculated from element center.
+     * Performs a slight user defined diagonal movement of element to avoid limitations of "distance"
+     * More info: https://github.com/clauderic/react-sortable-hoc
+     * @param element element to drag and drop.
+     * @param distance integer horizontal and vertical offsets to move element (calculated from given element center).
+     * @param xOffset horizontal integer offset destination for dropping (calculated from given element center).
+     * @param yOffset vertical integer offset destination for dropping (calculated from given element center).
+     */
+    public void dragWithDistanceAndDropToOffsetXY(WebElement element, int distance, Integer xOffset, Integer yOffset) {
+        getActions().clickAndHold(element)
+                .moveByOffset(distance, distance)
+                .moveByOffset(xOffset - distance, yOffset - distance)
+                .release().perform();
+    }
+
+    /**
+     * Simulates a drag from source react element and drop to target element
+     * Performs a slight user defined delay before drag to avoid limitations of "pressDelay"
+     * More info: https://github.com/clauderic/react-sortable-hoc
+     * @param source element to start the drag
+     * @param delay long to define how much to delay in millis after clickAndHold
+     * @param xOffset horizontal integer offset destination for dropping (calculated from given element center).
+     * @param yOffset vertical integer offset destination for dropping (calculated from given element center).
+     */
+    public void dragWithDelayAndDropToOffsetXY (WebElement source, long delay, Integer xOffset, Integer yOffset) {
```

Comment:
```suggestion
    public void dragWithDelayAndDropToOffsetXY(WebElement source, long delay, Integer xOffset, Integer yOffset) {
```

Reply:
Thanks for response,

I agree with your ideas to be configurable and use one method, but the problem i see is backwards compatibility for the ones, that already use html5DragAndDrop (if we merge it with dragAndDrop).
Also, I believe there can be projects that have different UI elements which work with limitations and the ones, that work with simple drag and drop.
From the selenium actions perspective, both elements should work with drag and drop used with limitation, as drag and drop with limitation only adds additional actions (moveByOffset or pause), but html5draganddrop is different story.

My suggestion
Leave 2 methods:
DragAndDropTo (would try/return simple or distance or delay)
Html5dragAndDrop

Will think of something and commit.